scripts/run_tracker.py

```
@@ -0,0 +1,12 @@
+from dreem.inference import track
+from omegaconf import OmegaConf
+import os
```

Ruff

`os` imported but unused

Remove unused import: `os`

(F401)